### DISCUSSION TITLE
alevin library type for 10x Chromium v3

### DISCUSSION THREAD

**Discussion by deevdevil88** [COMBINE-lab/salmon#674] [Q&A (single-cell specific)]
Hi,
i just wanted confirm if the library type for alevin quant for 10x chromium 3' v3 is still ISR?

Best,
Devika

**Comment by k3yavi** [comment]
Yep, that's correct.

**Reply by lianos** [reply]
Is there maybe a mistake in the "[improving-txome-specificity](https://github.com/COMBINE-lab/salmon/discussions/674)" alevin-fry tutorial, then?

The current command library type specified to align the `pbmc_10k_v3` reads is `-l IU` instead of `-l ISR`:

```bash
salmon alevin -i grch38_splici_idx -p 16 -l IU --chromiumV3 --sketch \
-1 pbmc_10k_v3_fastqs/pbmc_10k_v3_S1_L001_R1_001.fastq.gz \
pbmc_10k_v3_fastqs/pbmc_10k_v3_S1_L002_R1_001.fastq.gz \
-2 pbmc_10k_v3_fastqs/pbmc_10k_v3_S1_L001_R2_001.fastq.gz \
pbmc_10k_v3_fastqs/pbmc_10k_v3_S1_L002_R2_001.fastq.gz \
-o pbmc_10k_v3_map \
--tgMap t2g.tsv
```

**Reply by rob-p** [reply]
Thanks for the feedback @lianos.  The reason thats not a typo is that in the alevin-fry pipeline, unlike alevin, the filtering is done in the "generate permit list" and collate steps.  Thus, when you specify `-d fw` in the `generate-permit-list` command you are telling alevin-fry to keep only those mappings in the forward orientation.  Alevin doesn't have an independent filtering command, so it must all be enforced via the library type.

**Reply by lianos** [reply]
Ohh ... I see, thanks for the explanation!
